services/api-server/src/simcore_service_api_server/exceptions/backend_errors.py

```diff
@@ -0,0 +1,99 @@
+from fastapi import status
+
+from ._base import ApiServerBaseError
+
+
+class BackEndError(ApiServerBaseError):
+    """Base class for all backend exceptions"""
+
+    status_code = status.HTTP_502_BAD_GATEWAY
+
+
+class ListSolversOrStudiesError(BackEndError):
+    msg_template = "Cannot list solvers/studies"
+    status_code = status.HTTP_404_NOT_FOUND
```

Comment:
It is not obvious that this `status_code` corresponds to the one you are going to map to in the api-server. 
Doc a bit and/or perhaps rename it... `to_status_code` or `to_api_server_status_code`? or something analogous

Reply:
documented it